File: barcosEnContienda/src/gestorAplicacion/Items/Objeto.java
package gestorAplicacion.Items;

//Clase para la creacion de objetos
public class Objeto {
	public String nombre;
	public String descripcion;
	
	
	public Objeto(String nombre, String descripcion) {
		this.nombre=nombre;
		this.descripcion=descripcion;
	}
	public void Usar() {}
	public static void destruir(Objeto objeto) {
		objeto.nombre = "xxxxx";
		objeto.descripcion = "xxxxx";
	}
}

File: practica-2-grupo-1-equipo-7/src/gestorAplicacion/items/objeto.py
class Objeto:
    def __init__(self,nombre, descripcion):
        self._nombre=nombre
        self._descripcion=descripcion
        
    def destruir(self):
     
     self._nombre = ""
     self._descripcion = ""
    
    def getNombre(self):
        return self._nombre
    
    def getDescripcion(self):
        return self._descripcion